.travis.yml

```diff
@@ -14,9 +14,10 @@ env:
     - secure: "AfE+PLANOT0mgUKpgGUu4yIrSNKxoNST5zFBeajONEEE0wl/bZVE6iGqucNPR0EaIgMfk+NGgCLLM7VFfy+HF5alBiqUUOapef+fc91iHVzoBgtlLXOO7E1UntrieBQCodURTaBtotCMe7jtlkNajkvUD6atKTYh1AYJ3JY2bHA="
     - GLIUM_HEADLESS_TESTS=1
   matrix:
+    - FEATURES='' TEST=0
     - FEATURES='headless' TEST=1
```

Comment:
Well, yeah. 99% of the tests create a headless context and run OpenGL commands. If you remove glutin, there's nothing left to create the context.